src/components/theme/styles.js

```diff
@@ -29,6 +29,7 @@ export const theme = {
     ...DefaultTheme.fonts,
     default: 'Roboto',
     slab: 'Roboto Slab',
+    medium: 'Roboto-Medium',
```

Comment:
i'm not sure we have this font.
I think we have Roboto with fontWeight:500 for medium and fontWeight:700 for bold

Reply:
It was in already in code. I've just move it to theme config

Reply:
so we need to change it. we dont have it anymore.
we use robot with 500 for medium and 700 for bold

Reply:
ok I will change it